From: Richard B Winters <rik@mmogp.com>
Date: Sun, 3 May 2015 16:39:55 -0400
Subject: Fix conf directory in pcsd init

 - The conf file is set to be read from /etc/sysconfig/pcsd, 
   we'll change that to be /etc/init/pcsd.conf instead

Change-Id: I9a5fa9f40b5c470b35adabf4d718730a366d1a5c
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/pcsd | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/pcsd/pcsd
+++ b/pcsd/pcsd
@@ -25,9 +25,10 @@
 
 exec="/usr/bin/ruby"
 prog="pcsd"
+extension=".conf"
 config="/usr/share/pcsd"
 
-[ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
+[ -e /etc/init/$prog$extension ] && . /etc/init/$prog$extension
 
 lockfile=/var/lock/subsys/$prog
 
@@ -36,7 +37,7 @@
     echo -n $"Starting $prog: "
     #export GEM_HOME=/usr/lib/pcsd/vendor/bundle/ruby
     cd /var/lib/pcsd
-    daemon --command='/usr/bin/ruby -I/usr/share/pcsd /usr/share/pcsd/ssl.rb'
+    daemon --name=pcsd --command='/usr/bin/ruby -I/usr/share/pcsd /usr/share/pcsd/ssl.rb'
     # if not running, start it up here, usually something like "daemon $exec"
     retval=$?
     echo
